README.md

```diff
@@ -1,3 +1,38 @@
-# Moeller Lab metagenomics processing pipeline
+# Moeller Lab Metagenomics Processing Pipeline
 
+## Overview
 Snakemake pipeline for basic processing of metagenomic data from the lab.
+
+Module 1: **sn-mg-QC**
+
+Inputs:
+  - Directory of raw fastq files from samples
+  - `File_Paths.txt` document, with 3 columns: `Sample_ID`, `R1`, `R2`
+    - Files will be renamed with `Sample_ID`
+    - `R1` and `R2` should be the full path to the forward and reverse reads, respectively
```

Comment:
Wondering if we should add an additional layer with a notion of "sequencing unit", allowing us to include reads from multiple runs.

Reply:
Do you mean being able to integrate reads from the same sample that have been sequenced more than once? How would this be handled -- by just concatenating them together?